spec/triggers/prometheus.md

```diff
@@ -0,0 +1,17 @@
+# Prometheus Trigger
+
+This specification describes the `prometheus` trigger.
+
+```yaml
+  triggers:
+  - type: prometheus
+    metadata:
+      serverAddress: <Prometheus server URL> #Required
+      metricName: <name of metric> #Required
+      threshold: '<threshold at which auto-scale is triggered>' #Required
+      query: <PromQL query> # Note: query must return a vector/scalar single element response
+```
+
```

Comment:
Can you add a small description here on what the values are expected to be please?

Something similar to this: https://github.com/kedacore/keda/blob/master/spec/triggers/azure-service-bus.md

Should tell new folks how to configure it correctly, can be very basic.

Reply:
Looks great @abhirockzz - yeah if you could just add one filled out example that has like the format of each of these.  If not I can look at this when I write other docs for prometheus and potentially add to your PR.  Thanks a ton for contributing.

Reply:
Done!